### PR TITLE
fix typo in Mobile accessibility

### PR DESCRIPTION
**Pull request by yin1999**
- Add code block highlight.
- Update links.

### CODE REVIEW

**Review by JuanVqz — APPROVED** [review]
This looks good to me, @mdn/yari-content-es is anyone available to double-check this, please?

File: files/es/learn/accessibility/mobile/index.html [hunk 1]
```diff
@@ -286,36 +249,32 @@ <h4 id="No_deshabilitar_el_zoom">No deshabilitar el zoom</h4>
...
 
-<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#Control mechanisms">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
+<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#mecanismos_de_control">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```
Comment (JuanVqz) [inline files/es/learn/accessibility/mobile/index.html:260]: So, should this be like this? 
```suggestion
<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#mecanismos_de_control">mecanismos de control</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```

Comment (yin1999) [reply files/es/learn/accessibility/mobile/index.html]: Applied this.

**Comment by Graywolf9** [comment]
Thank you so much @yin1999 !

**Pull request merged**